lib/query_components/text_query.rb

```diff
@@ -51,6 +63,12 @@ def all_searchable_text_query
       dis_max_query(queries, tie_breaker: 0.1)
     end
 
+    def quoted_phrase_conditions
+      groups = [ field_boosts_phrase ]
+      dismax_groups(groups)
```

Comment:
Sorry - disregard this, I misread the code.  The "field_boosts_phrase" bit consists of multiple items which dismax_groups joins together with a dismax query.